cpp/src/arrow/csv/options.h

```diff
@@ -166,6 +166,9 @@ struct ARROW_EXPORT ReadOptions {
   /// Create read options with default values
   static ReadOptions Defaults();
 
+  /// weather to use an IO thread to read ahead
+  bool use_read_ahead = false;
```

Comment:
Is there any reason to make this false by default?

Reply:
Also, probably name this `use_readahead` ("readahead" is a noun in this context)

Reply:
Done